stdlib/Random/docs/src/index.md

```diff
@@ -198,8 +195,6 @@ end
 ```
 and that we *always* want to build an a alias table, regardless of the number of values needed (we learn how to customize this below). The methods
 ```julia
-Random.gentype(::Type{<:DiscreteDistribution}) = Int
```

Comment:
Here it may be better to replace `gentype` by `eltype`.

Reply:
Ok, adding back.